client/lib/abtest/active-tests.js

```diff
@@ -125,5 +125,14 @@ module.exports = {
 		defaultVariation: 'showPlansAfterAuth',
 		allowExistingUsers: true
 	},
-};
 
+	presaleChatButton: {
+		datestamp: '20161129',
+		variations: {
+			showChatButton: 80,
+			original: 20
```

Comment:
Maybe rename here and below to `noChatButton` or something more descriptive than `original`? I'm not sure that's essential, and if using something like `original` is how things are done elsewhere, that's fine. It just occurred to me that reading through the code and imagining trying to look at data for this, `original` isn't super meaningful over time.